deltacat/utils/pyarrow.py

```diff
@@ -253,6 +254,41 @@ def s3_file_to_table(
         f"Reading {s3_url} to PyArrow. Content type: {content_type}. "
         f"Encoding: {content_encoding}"
     )
+
+    if content_type == ContentType.PARQUET.value:
+        from daft.table import Table
+        from daft import TimeUnit
+        from daft.io import IOConfig, S3Config
+
+        kwargs = {}
+        if pa_read_func_kwargs_provider is not None:
+            kwargs = pa_read_func_kwargs_provider(content_type, kwargs)
+
+        coerce_int96_timestamp_unit = TimeUnit.from_str(
+            kwargs.get("coerce_int96_timestamp_unit", "ms")
+        )
+
+        table = Table.read_parquet(
+            path=s3_url,
+            columns=include_columns,
+            coerce_int96_timestamp_unit=coerce_int96_timestamp_unit,
+            io_config=IOConfig(
+                s3=S3Config(
+                    key_id=s3_client_kwargs["aws_access_key_id"],
+                    access_key=s3_client_kwargs["aws_secret_access_key"],
+                    session_token=s3_client_kwargs["aws_session_token"],
+                    num_tries=25,
+                )
+            ),
```

Comment:
Can we allow passing rest of the kwargs to `read_parquet` directly, so that we can modify io_config etc. without requiring a DeltaCAT change?